lib/pg_query/parse.rb

```diff
@@ -264,6 +264,10 @@ def load_objects! # rubocop:disable Metrics/CyclomaticComplexity
               function: next_item.func_call.funcname.map { |f| f.string.str }.join('.'),
               type: :call
             }
+          when :case_expr
+            subselect_items.concat(next_item.case_expr.args.map { |arg| arg.case_when.expr })
+            subselect_items.concat(next_item.case_expr.args.map { |arg| arg.case_when.result })
```

Comment:
Minor: I got curious if the creation of an intermediary Array here was less efficient than `<<`ing many times onto subselect_items.

In all: Cool feature addition!